.gitignore

```diff
@@ -2,3 +2,4 @@ build
 composer.lock
 docs
 vendor
+.idea
```

Comment:
Remove this line, it should be placed in your own global .gitignore.

Reply:
done